service/src/main/java/bio/terra/externalcreds/terra/FirecloudOrchestrationClient.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.externalcreds.terra;
+
+import bio.terra.externalcreds.ExternalCredsException;
+import bio.terra.externalcreds.config.ExternalCredsConfig;
+import bio.terra.externalcreds.util.EcmRestTemplate;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class FirecloudOrchestrationClient {
+
+  private final ExternalCredsConfig externalCredsConfig;
+  private final EcmRestTemplate restTemplate;
+  private final HttpHeaders headers;
+
+  public FirecloudOrchestrationClient(
+      ExternalCredsConfig externalCredsConfig, EcmRestTemplate restTemplate) {
+    this.externalCredsConfig = externalCredsConfig;
+    this.restTemplate = restTemplate;
+    this.headers = new HttpHeaders();
+    headers.setAccept(List.of(MediaType.APPLICATION_JSON));
```

Comment:
Can `headers` just be private static or just lazy loaded?  Looks like we aren't changing them ever.